spec/serializers/user_bookmark_list_serializer_spec.rb

```diff
@@ -29,5 +29,16 @@ def run_serializer
         %w[UserTestBookmarkSerializer UserTopicBookmarkSerializer UserPostBookmarkSerializer],
       )
     end
+
+    it "serializes categories" do
+      topic_category = Fabricate(:category)
+      topic_bookmark.bookmarkable.update!(category: topic_category)
+      post_category = Fabricate(:category)
+      post_bookmark.bookmarkable.topic.update!(category: post_category)
+
+      serializer = run_serializer
+
+      expect(serializer.categories).to contain_exactly(topic_category, post_category)
```

Comment:
I would have thought we'd need to set something up with `SiteSetting.lazy_load_categories_groups_map` here, since the serializer is checking `scope.can_lazy_load_categories?`. Is that context enabled by default in specs?

Reply:
This is a bit weird. Rails will call `include_` properties to determine if a property is serialized or not, but we are calling that directly, bypassing `include_` method.